.gitignore

```diff
@@ -119,6 +119,7 @@ scripts/nodes.txt
 /doc/_build
 /doc/source/_static/thumbs
 /doc/source/tune/generated_guides/
+/doc/source/**/doc/
```

Comment:
@justinvyu can you explain why this is necessary?

Reply:
We now have autogenerated API files under `/doc/source/tune/api/doc/*`, `/doc/source/data/api/doc/*`, etc. We shouldn't track these with git.